addons/bestja_frontend_fpbz/templates/snippets/quote_div.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+        <template id="quote_div_snippet" inherit_id="website.snippets" name="FPBZ snippet with big quote">
+            <xpath expr="//div[@id='snippet_content']" position="inside">
+                <div>
+                    <div class="oe_snippet_thumbnail">
+                        <img class="oe_snippet_thumbnail_img" src="/bestja_frontend_fpbz/static/src/img/snippets/s_block_quote.png"/>
+                        <span class="oe_snippet_thumbnail_title">Big quote</span>
+                    </div>
+                    
+                    <section class="oe_snippet_body">
+                        <div class="container-fluid quote_div">
+                            <div class="row">
+                                <div class="col-md-offset-1 col-md-10">
+                                    <blockquote>
+                                        <span class="big_apostrophe_before">”</span>
```

Comment:
Although it may be displayed nicely on the webpage, from the semantical point of view this is closing quotation mark (similar can be said about the mark on the end of the blockquote). Also, since we are inside the blockquote anyway, putting quotation mark here is semantically redundant. Can we add these quotation marks via a pseudo-class? This will solve both problems, since in CSS it isn’t very important whether we use the correct character to mark the quotation.